moveit_core/kinematic_constraints/src/kinematic_constraint.cpp

```diff
@@ -951,7 +955,7 @@ void VisibilityConstraint::getMarkers(const moveit::core::RobotState& state,
   mk.pose.orientation.y = 0;
   mk.pose.orientation.z = 0;
   mk.pose.orientation.w = 1;
-  mk.lifetime = rclcpp::Duration(60);
+  mk.lifetime = rclcpp::Duration(std::chrono::seconds(60));
```

Comment:
I agree with Henning about these.  I think doing it like this is nicer:

```suggestion
  mk.lifetime = rclcpp::Duration::from_seconds(60);
```